packages/editor-ui/src/stores/ui.store.ts

```diff
@@ -329,18 +328,29 @@ export const useUIStore = defineStore(STORES.UI, {
 			};
 		},
 		upgradeLinkUrl() {
-			return (source: string, utm_campaign: string): string => {
-				const linkUrlTranslationKey = this.contextBasedTranslationKeys
```

Comment:
can we clean this up then if we are not using them?

Reply:
We are still using contextBasedTranslationKeys in a bunch of places!

Reply:
I meant specifically `.upgradeLinkUrl` there

Reply:
ahhhh gotcha!